pkg/tracer/config/config.go

```diff
@@ -40,7 +57,11 @@ func MakeCommonConfig() *CommonConfig {
 		MaxConnections:            10000,
 		UDPConnTimeout:            30 * time.Second,
 		FilterInactiveConnections: true,
-		HttpMetricPrecision:       0.01,
-		EnableTracepipeLogging:    false,
+		HttpMetricConfig: HttpMetricConfig{
+			SketchType: CollapsingLowest,
+			Accuracy:   0.01,
+			MaxNumBins: 32,
```

Comment:
I thought we would use a default of 1024?

Reply:
Fixed!